### PR TITLE
Postgres chained concatenate expression

### PR DESCRIPTION
**Pull request by billy1624**

### CODE REVIEW

**Comment by billy1624** [comment]
Do you think having a alias `concat` for `concatenate` would be good? @tyt2y3

**Comment by tyt2y3** [comment]
> Do you think having an alias `concat` for `concatenate` would be good? @tyt2y3

Sounds nice!

**Comment by billy1624** [comment]
> Do you think having an alias `concat` for `concatenate` would be good? @tyt2y3

Added

File: src/expr.rs [hunk 1]
```diff
@@ -1733,4 +1743,44 @@ impl SimpleExpr {
...
+    pub fn concatenate<T>(self, right: T) -> Self
+    where
+        T: Into<SimpleExpr>,
+    {
+        self.binary(BinOper::Concatenate, right.into())
+    }
+
+    /// Alias of [`SimpleExpr::concatenate`]
+    #[cfg(feature = "backend-postgres")]
+    pub fn concat<T>(self, right: T) -> Self
```
Comment (tyt2y3) [inline src/expr.rs:1780]: Yes. an alias is nice

**Review by tyt2y3 — APPROVED** [review]

**Pull request merged**